### ISSUE TITLE
[temp.deduct.call] wording

### ISSUE THREAD

**Issue by BRevzin** [cplusplus/draft#517]
Paragraph 3 reads:

"If P is a cv-qualified type, the top level cv-qualifiers of P's type are ignored for type deduction. If P is a reference type, the type referred to by P is used for type deduction."

The wording could be improved to make it clearer that the two sentences are mutually exclusive. Perhaps "Otherwise, if P is a reference type, ..." Or potentially reversing the two sentences: "If P is a reference type.... Otherwise, if P is a cv-qualified type, ..."

**Comment by jwakely** [comment]
It's not strictly necessary, because references cannot be cv-qualified.

**Comment by dyp-cpp** [comment]
It caused some confusion [here](http://stackoverflow.com/questions/31292403/template-deduction-interesting-case-c#31293774) (in Barry's and Anton Savin's answer), so I suggested it might be clarified. Barry originally thought the removal of cv-qualification causes type deduction to recur with the non-cv-qualified type; Anton might have though both sentences apply as well -- in any case, both seem to have though that `P == T const&` will get stripped to `P == T` instead of `P == T const`. That's why I suggested it might be worthwhile to clarify it.
